modules/stochastic_tools/include/vectorpostprocessors/PolyChaosSobolStatistics.h

```diff
@@ -0,0 +1,70 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+// MOOSE includes
+#include "GeneralVectorPostprocessor.h"
+#include "PolynomialChaos.h"
+
+class PolyChaosSobolStatistics;
```

Comment:
```suggestion
class PolynomialChaosSobolStatistics;
```
Can you do this so it is consistent with the `PolynomialChaos` model name. Hopefully, before too long we will have some wrapper actions that control all this stuff.